cmd/process-agent/collector.go

```diff
@@ -217,12 +217,12 @@ func (l *Collector) run(exit chan struct{}) error {
 
 	processForwarderOpts := forwarder.NewOptions(api.KeysPerDomains(l.cfg.APIEndpoints))
 	processForwarderOpts.DisableAPIKeyChecking = true
-	processForwarderOpts.RetryQueueSize = l.cfg.QueueSize // Allow more in-flight requests than the default
+	processForwarderOpts.SetRetryQueuePayloadsTotalMaxSizeFromQueueMax(l.cfg.QueueSize) // Allow more in-flight requests than the default
```

Comment:
Shouldn't we use `ProcessQueueBytes` directly instead? https://github.com/DataDog/datadog-agent/blob/06b119477628dcaf6e174cbc51c002e14f8d6d21/pkg/process/config/config.go#L190